### PR TITLE
New placement for color box (#866)

### PR DESCRIPTION
**Pull request by stefaweb**
27292291-5119892e-5513-11e7-8e5c-d13f15c9218f

### CODE REVIEW

**Comment by stefaweb** [comment]
Colors change a little in color box when mouse over.

capture d ecran 2017-06-19 a 21 28 54
capture d ecran 2017-06-19 a 21 28 45

**Comment by stefaweb** [comment]
Over. Can be merged if ok.

File: mailscanner/functions.php [hunk 1]
```diff
@@ -372,7 +372,6 @@ function html_start($title, $refresh = 0, $cacheable = true, $report = false)
     echo '</td>' . "\n";
 
     echo '<td align="left" valign="top">' . "\n";
-    printColorCodes();
     echo '  </td>' . "\n";
```
Comment (endelwar) [inline mailscanner/functions.php]: This empty td can be removed, beware of the column count on other places of the code

Comment (stefaweb) [inline mailscanner/functions.php]: Done.

**Comment by stefaweb** [comment]
Is it ok this time?

**Pull request merged**